pennylane/ftqc/pauli_tracker.py

```diff
@@ -99,15 +110,15 @@ def xz_to_pauli(x: np.uint8, z: np.uint8) -> Operator:
     raise ValueError("x and z should either 0 or 1.")
 
 
-def pauli_prod(ops: List[Operator]) -> Operator:
+def pauli_prod(ops: List[Operator]) -> Tuple[np.uint8, np.uint8]:
     """
     Get the result of a product of a list of Pauli operators. The result is a new Pauli operator up to a global phase.
 
     Args:
         ops (List[qml.operation.Operator]): A list of Pauli operators with the same target wire.
 
     Return:
-        A new Pauli operator.
+        A xz tuple representing a new Pauli operator.
```

Comment:
Do we handle an explicit conversion of the 0/1 values in the module-global lists to np.uint8?
It looks as though we are returning 64-bit integers for 0/1 but saying in the type info we are returning NP uints. 

We can either fix the data stored in the top-of-file stores, or just return the ints themselves as the type.

Reply:
Nice catch @mlxd.